docs/fundamentals/connection/connect-to-mongodb.txt

```diff
@@ -0,0 +1,334 @@
+     - Specifies the database driver to use for the connection.
+
+   * - ``dsn``
+     - The data source name (DSN) that specifies the MongoDB connection URI.
+
+   * - ``host``
+     - Specifies the network address of one or more MongoDB nodes in a
+       deployment. You can use this setting instead of the ``dsn`` setting.
+
+   * - ``database``
+     - Specifies the name of the MongoDB database to read and write to.
+
+   * - ``username``
+     - Specifies your database user's username credential to authenticate
+       with MongoDB.
+
+   * - ``password``
+     - Specifies your database user's password credential to authenticate
+       with MongoDB.
+
+   * - ``options``
+     - Specifies connection options to pass to MongoDB that determine the
+       connection behavior.
+
+   * - ``driverOptions``
+     - Specifies options specific to pass to the MongoDB PHP Library driver
+       that determine the driver behavior for that connection.
+
+.. note::
+
+   You can specify the following settings in the ``dsn`` configuration
+   as parameters in your MongoDB connection string instead of as array items:
+
+   - ``host``
+   - ``username``
+   - ``password``
+   - ``options`` and ``driverOptions``, which are specified by the option name
+
+The following example shows how you can specify your MongoDB connection details
+in the ``connections`` array item:
+
+.. code-block:: php
+   :caption: Example config/database.php MongoDB connection configuration
+
+   'connections' => [
+       'mongodb' => [
+           'driver' => 'mongodb',
+           'dsn' => 'mongodb+srv//myUser:myPass123@sample.host:27017/',
+           'database' => 'sample_mflix',
+           'options' => [
+               'maxPoolSize' => 20,
+               'w' => 'majority',
+           ],
+           'driverOptions' => [
+               'serverApi' => 1,
+           ],
+       ],
+       // ...
+   ],
+
+The following sections provide common ways of specifying MongoDB connections.
+
+.. _laravel-atlas-connection-example:
+
+Connection Example
+------------------
+
+This section shows how to configure your Laravel application's DSN by using a
+MongoDB Atlas connection string.
+
+To add your MongoDB DSN to your Laravel application, make the following changes:
+
+- Add the DSN as an environment variable in your project's ``.env`` environment
+  configuration file. Set the variable value to your Atlas connection string.
+- Add a connection entry for your MongoDB connection in the ``connections``
+  array of your ``config/database.php`` configuration file. Set the ``dsn``
+  value of the connection entry to reference the environment variable that
+  contains your DSN.
+
+The following examples show how to specify ``"mongodb+srv://myUser:myPass123@mongodb0.example.com/"``
+as the connection string in the relevant configuration files:
+
+.. code-block:: php
+   :caption: Sample .env environment configuration
+
+   DB_URI=""mongodb+srv://myUser:myPass123@mongodb0.example.com/";
```

Comment:
```suggestion
   DB_URI="mongodb+srv://myUser:myPass123@mongodb0.example.com/"
```

Reply:
Whoops, great catch!
I should also use "bash" instead of "php" highlighting.